ubuntu-server-netboot.py

```diff
@@ -251,7 +255,12 @@ def cleanup(directory):
     )
     os.mkdir(os.path.join(staging_dir, "grub"))
     with open(os.path.join(staging_dir, "grub", "grub.cfg"), "w") as grub_cfg:
-        netboot_args = "%s url=%s" % (Netboot_Args, args.url)
+        if args.autoinstall_url:
+            autoinstall_args = f'"ds=nocloud-net;s={args.autoinstall_url}"'
+            netboot_args = "%s url=%s autoinstall %s" % (Netboot_Args, args.url, autoinstall_args)
+        else:
+            netboot_args = "%s url=%s" % (Netboot_Args, args.url)
+
```

Comment:
remember to also do the same for pxelinux - autoinstall isn't specific to UEFI.

Reply:
Good catch! Will do in the upcoming PR update.